### PR TITLE
Utype/polishing

### PR DESCRIPTION
**Pull request by lmichel**
Answer to 3 items of  #150 
no longer talk about about drawbacks
Mention the impossibility of doing joins
Rephrase item 3) as @Bonnarel suggested

### CODE REVIEW

**Comment by lmichel** [comment]
I couldn't assign the PR to @Bonnarel. This comment is ti make sure he will get a notification.

File: doc/introduction.tex [hunk 1]
```diff
@@ -18,22 +18,25 @@
...
+  \item UTypes constrain to single-table serializations. They do no allow to join data from different tables. 
   \item UTypes do not support annotating the VOTable content towards multiple models 
   (as TimeSeries or Catalog/Source for instance)
 \end{itemize}
 
-The UType approach could have been extended to overcome these drawbacks, but it has been decided to move forward a solution closer to the \texttt{VODML} \citep{2018ivoa.spec.0910L} concepts. 
+The UType approach could have been extended to overcome these limitations, but it has been decided to move forward a solution closer to the \texttt{VODML} \citep{2018ivoa.spec.0910L} concepts. 
 \texttt{VODML} is a meta-model that gives a standard way to express VO models as machine-readable XML document.
-In \texttt{VODML},  model leaves are no longer identified by simple strings like UTypes do, but by the role they play in a given location in the model hierarchy.
+In \texttt{VODML}, model leaves role are no longer given by simple strings like UTypes, but by the local role they play in the model hierarchy
```
Comment (mcdittmar) [inline doc/introduction.tex:37]: + In \texttt{VODML}, the role of model leaves are no longer given by simple strings like UTypes, but by the local role they play in the model hierarchy

**Review by mcdittmar — CHANGES_REQUESTED** [review]
* appendix_z.tex
    * I don't understand this.. not UTypes related, and appears to have a local path to the file.
* introduction.tex
    * just a phrasing change suggestion.

File: doc/introduction.tex (hunk 1 above)
Comment (lmichel) [reply doc/introduction.tex]: MCD's text replaces mine

**Comment by lmichel** [comment]
appendix_z.tex was there because I started working on the snippet from the wrong branch

File: doc/introduction.tex (hunk 1 above)
Comment (Bonnarel) [reply doc/introduction.tex]: OK

**Comment by Bonnarel** [comment]
>  UTypes are path-like strings (\texttt{m:a.b.c}) identifying model leaves. As there is no common rule to build UTypes,  each data model (e.g. Characterization, Obscore)  must come with its specific lists of UTypes. 

When used as \texttt{FIELD} attributes, UTypes allow one to connect particular columns to model leaves. This mapping method fits very well with the VOTable schema since it does not require any specific XML elements. 
% It has however a number of limitations that have been discussed many times:
However, some UTypes features that could complicate the mapping process have been discussed many times:

\begin{itemize}
  \item UTypes are labels attached to  a \texttt{FIELD} or \texttt{PARAM} that tells its role in the 
  model structure, typically a non cyclic graph.

**Pull request merged**